modules/imgproc/src/canny.cpp

```diff
@@ -805,6 +865,20 @@ void Canny( InputArray _src, OutputArray _dst,
 
     Mat src = _src.getMat(), dst = _dst.getMat();
 
+    CV_OVX_RUN(
+        false && /* disabling due to accuracy issues */
```

Comment:
Look at the following example:
(source, canny-cv-canny-vx-diff)
![canny-img-cv-vx-diff](https://cloud.githubusercontent.com/assets/2344952/20891892/2d58c5d0-bb1d-11e6-9725-e256d5ca65c3.png)